internal/checker/terraform/rules/git/github_actions_environment_secret_actions_secret_encrypted.rego

```diff
@@ -0,0 +1,61 @@
+# METADATA
+# title: "Verify that the GitHub Actions secrets are securely encrypted"
+# description: "Verify that the secrets used in GitHub Actions are adequately encrypted to safeguard sensitive information."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/integrations/github/latest/docs/resources/actions_environment_secret
+# custom:
+#   id: CB_TFGIT_004
+#   severity: HIGH
+package lib.terraform.CB_TFGIT_004
+
+import future.keywords.in
+
+isvalid(block) {
+	supported_resources := [
+		"github_actions_environment_secret",
+		"github_actions_organization_secret",
+		"github_actions_secret",
+	]
+	block.Type == "resource"
+	some label in block.Labels
+	label in supported_resources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[block] {
```

Comment:
```
fail[block] {
	some block in the input
	isvalid(block)
	block.Attributes.plaintext_value != ""
}

pass[block] {
	some block in input
	isvalid(block)
	not fail[block]
}
```

the plaintext_value attribute is optional based on the documentation. Requesting to put this in a fail block, to check if the attribute exists on the terraform file.

if you put it on the pass block, the policy always fails if the attribute does not exist on the terraform file.